### ISSUE TITLE
Implement the sign-in manager

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#43]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

The app-level sign-in manager class.

### Why

To handle sign-in status.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
